### PR TITLE
Updated the document to reflect the updates to the rule (up to 10 rul…

### PR DESCRIPTION
**Pull request by mpetersoCode55**
Updated the document to reflect the fact that you can have up to 10 rules associated with each token handler now.

### CODE REVIEW

**Review by oscarsernarosero — CHANGES_REQUESTED** [review]
Cool man. You still need to update the ##Other Functions section. I just noticed it says "In Application Handler" but it should say "in Asset Handler", and update the getters and setters listed there with the new ones you created.

**Review by oscarsernarosero — APPROVED** [review]
Looks great now Michael.

**Review by ShaneDuncan602 — APPROVED** [review]
Great job.

File: docs/userGuides/rules/ALLOW-DENY-ORACLE.md [hunk 1]
```diff
@@ -84,6 +84,14 @@ error AddressIsDenied();
 
 The selector for this error is `0x2767bda4`.
 
+When adding an oracle rule to a token, if there are already 10 oracle rules associated the handler will rever with the following error:
```
Comment (TJ-Everett) [inline docs/userGuides/rules/ALLOW-DENY-ORACLE.md]: revert*

**Review by TJ-Everett — APPROVED** [review]
Just one tiny typo - otherwise looks great!

File: docs/userGuides/rules/ALLOW-DENY-ORACLE.md (hunk 1 above)
Comment (mpetersoCode55) [reply docs/userGuides/rules/ALLOW-DENY-ORACLE.md]: Resolved

**Pull request merged**